Userland/Services/DHCPClient/main.cpp

```diff
@@ -40,7 +40,7 @@
 static u8 mac_part(const Vector<String>& parts, size_t index)
 {
     auto chars = parts.at(index).characters();
-    return (chars[0] - '0') * 16 + (chars[1] - '0');
+    return strtoul(chars, nullptr, 16);
```

Comment:
Maybe use `StringUtils::convert_to_uint_from_hex()` instead of `strtoul`?

Reply:
done :)

Reply:
Commit message still references `strtoul` ;)

Reply:
whoops! fixed

Reply:
whoops! fixed